app/internal/page/share_code.go

```diff
@@ -7,30 +7,43 @@ import (
 	"github.com/ministryofjustice/opg-modernising-lpa/internal/notify"
 )
 
+var useTestCode = false
```

Comment:
we need to come up with a different way of doing this (and the one in `random`) as it will be set `true` until the app is restarted, making using the app "properly" impossible if someone uses testing-start

Reply:
Is the issue that we inject `sharecode` at run time and then can't change it? I'd hoped by having the property here (and assuming we're passing round a pointer) we could update it.

Maybe we need `Send()` to always reset `useTestCode` back to false after it's been set to true, like:

```golang
var shareCode string

if useTestCode {
    shareCode = "abcdef123456"
    useTestCode = false
} else {
    shareCode = s.randomString(12)
}
```

Reply:
yeah that should work for now